docs/changelog.rst

```diff
@@ -4,6 +4,10 @@ Changelog
 
 `CalVer, YY.month.patch <https://calver.org/>`_
 
+24.11.1
+======
+- :ref:`ASYNC100 <async100>` now ignores :func:`trio.open_nursery` and :func:`anyio.create_task_group` as cancellation sources.
```

Comment:
```suggestion
- :ref:`ASYNC100 <async100>` now ignores :func:`trio.open_nursery` and :func:`anyio.create_task_group` as cancellation sources,
  because they are schedule points but not cancellation points.
```

This is a subtle enough concept that we should not assume users are familiar with it - I'd also call out this distinction in the docs for `ASYNC100`, and create glossary entries for both which are linked from "checkpoint" and link in turn to Trio docs (eg for `cancel_shielded_checkpoint()` and `checkpoint_if_cancelled()`)

Reply:
good call, feel free to reword my attempt :)